linux/installers/apt.sh

```diff
@@ -4,7 +4,17 @@ set -euo pipefail
 echo 'APT::Update::Error-Mode "any";' | sudo tee /etc/apt/apt.conf.d/warnings-as-errors
 
 # Prevent kernel upgrades
+sudo apt-mark hold \
+  linux-generic \
+  linux-headers-generic \
+  linux-headers-virtual \
+  linux-image-generic \
+  linux-image-virtual
+  linux-virtual
 sudo apt-mark hold "$(uname -r)"
+if [ "${NV_RUNNER_ENV}" == "aws" ]; then
+  sudo apt-mark hold linux-aws linux-headers-aws linux-image-aws
+fi
```

Comment:
```suggestion
sudo apt-mark hold \
  linux-{aws,generic,virtual} \
  linux-headers-{aws,generic,virtual} \
  linux-image-{aws,generic,virtual} \
```

Can we simplify all of these commands to this suggestion?

From the screenshot below, `apt-mark $(uname -r)` doesn't match any packages. And the `aws`/`virtual` packages seems to be included in the default Ubuntu package repositories.

![image](https://github.com/user-attachments/assets/5642ff37-09c2-4554-ab8d-f1f5d69d93cb)

Reply:
`apt-mark $(uname -r)` is not matching any packages because you are using an empty docker image, which is not matching your kernel and doesn't have your kernel modules installed. It does match packages in VM images: https://github.com/nv-gha-runners/vm-images/actions/runs/10995676744/job/30527057765#step:7:73
But I'm fine with removing this

Reply:
let's keep it based on your log link